plugins/inputs/tail/tail.go

```diff
@@ -223,6 +226,21 @@ func (t *Tail) tailNewFiles(fromBeginning bool) error {
 				// we're already tailing this file
```

Comment:
I'd suggest to print `file` here, to check if the same pipe is tailed ore than once.

Reply:
> I'd suggest to print `file` here, to check if the same pipe is tailed ore than once.

@dshnayder , this keeps on flooding telegraf log as tailNewFiles function is keep on getting called multiple times

Reply:
How this function gets called multiple times? It supposed to be called once pr pipe, so its two times for you config.

Reply:
> How this function gets called multiple times? It supposed to be called once pr pipe, so its two times for you config.

@dshnayder , It keeps on getting called multiple times throughout telegraf lifetime. let me investigate on how. It does not gets called only twice. thats why we have line 225 and line 235 to check if we have already started adding the tail.